src/main/java/io/opentracing/contrib/common/Classes.java

```diff
@@ -51,21 +52,21 @@ private static void recurse(final Class<?> iface, final HashSet<Class<?>> set) {
    */
   public static Class<?>[] getAllInterfaces(final Class<?> cls) {
     Class<?> parent = cls;
-    Class<?>[] ifaces = null;
+    Class<?>[] ifaces;
```

Comment:
Removed the redundant initialization, since it will be initialized in the do while loop